src/game/boe.main.cpp

```diff
@@ -2,9 +2,12 @@
 #include "boe.global.hpp"
 #include "universe.hpp"
 
+// XXX now that we no longer use boost threads, do we still need this #define?
```

Comment:
This relates to my Mac (Xcode 4) build, but given that it specifically references Boost.Thread which we're no longer using, it's probably safe to remove it.

Reply:
done